vagrant/provisioning/roles/arkcase-file-utils/tasks/main.yml

```diff
@@ -62,3 +62,36 @@
     state: absent
     path: /tmp/en-US
   when: open_office_latest_version is not defined or not open_office_latest_version
+
+- name: Check Libreoffice Installation
+  become: yes
+  stat:
+    path: "/opt/libreoffice/program/soffice"
+  register: libreoffice_bin_exists
+
+- include_tasks: "{{ role_path }}/../common/tasks/download.yml"
+  loop:
+    - name: LibreOffice
+      dest: "/tmp/LibreOffice_{{ libreoffice_version | default('7.3.4') }}_Linux_x86-64_install-rpm_en-US.tar.gz"
+      owner: root
+      url: "https://ftp.osuosl.org/pub/tdf/libreoffice/stable/{{ libreoffice_version | default('7.3.4') }}/rpm/x86_64/LibreOffice_{{ libreoffice_version | default('7.3.4') }}_Linux_x86-64_rpm.tar.gz --no-check-certificate"
+
+- name: unpack LibreOffice RPM
+  become: yes
+  unarchive:
+    src: "/tmp/LibreOffice_{{ libreoffice_version | default('7.3.4') }}_Linux_x86-64_install-rpm_en-US.tar.gz"
+    remote_src: yes
+    dest: /tmp/
```

Comment:
These commands will run every time, which will be slow since LibreOffice is such a large package.  You could add a `when: not libreoffice_bin_exists.stat.exists`

Reply:
Thanks Dave. Just did.